tests/build/conftest.py

```diff
@@ -41,3 +42,41 @@ def test_gene():
         "vega_id": "1",
     }
     return gene
+
+
+@pytest.fixture
+def test_transcript():
```

Comment:
The two dicts are not the same. Missing from `tests/conftest.py` are: 'ensembl_transcript_id'
'transcript_start'
'transcript_end'
'hgnc_id'

All of these are tested in the new code in order to reach previously untouched lines. I think I copy/pasted my transcript from the logs.

-So what to do?
* Keep both
* Merge
* A lot of hands on

Is the correct representation of transcripts represented in a class or definition somewhere?

Reply:
I'd say merge, if possible? Does it work if you add the missing key/values to tests/conftest? Or replace it with the one one in this PR?

Reply:
I'm suspecting that the high-level transcript record doesn't reflect reality. I'll look into it today. (Using classes or well defined dicts, for central data structures would help to avoid problems like this)